.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Copied from neodarwin.  Without it, vscode was using a default that made a lot of changes.